recon_test_pack/lm_to_projdata.par

```diff
@@ -0,0 +1,44 @@
+lm_to_projdata Parameters:=
+
+  input file := ${INPUT}
+output filename prefix := ${OUT_PROJDATA_FILE}
+
+  ; parameters that determine the sizes etc of the output
+
+       template_projdata := ${TEMPLATE}
+    ; the next can be used to use a smaller number of segments than given 
```

Comment:
I think from here on all keywords are the defaults. I'd therefore recommend removing them. Better for future compatibility?